config/taxonomy/example/module/actions.yaml

```diff
@@ -1,6 +1,6 @@
 definitions:
   Action:
-    oneOf:
+    anyOf:
```

Comment:
Why the change? I believe that it must be oneOf.

Reply:
@roee88 `oneOf` was throwing this error for a [valid Fybrik module](https://github.com/fybrik/fybrik/blob/master/manager/testdata/unittests/implicit-copy-batch-module-csv.yaml):
``` 
INFO    test-module-controller  Fybrik module validation failed FybrikModule.app.fybrik.io "implicit-copy-batch-s3" is invalid: [spec.capabilities.0.actions.0: Invalid value: map[string]interface {}{"name":"RedactAction"}: Must validate one and only one schema (oneOf), spec.capabilities.0.actions.0: Invalid value: map[string]interface {}{"name":"RedactAction"}: RedactAction is required, spec.capabilities.0.actions.1: Invalid value: map[string]interface {}{"name":"RemoveAction"}: Must validate one and only one schema (oneOf), spec.capabilities.0.actions.1: Invalid value: map[string]interface {}{"name":"RemoveAction"}: RemoveAction is required]  {"FybrikModule": "fybrik-system/implicit-copy-batch-s3"}
```